playbooks/mariadb_gcp_dev.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: gcp_mariadb_dev
```

Comment:
I'd still like to see `name:` lines on all plays. The `name:` line has no requirements, it's just a title for the play so when you read the playbook or watch it execute you can easily understand the intent. For example, this one might be:
`- name: build a test db on gcloud`. Once you add `- name:`, the `hosts:` entry becomes the second line of the play and loses the `-`. 

Adding `- name:` to a play is similar to adding `- name:` to a task, but instead of describing what the task does, it describes what the entire play does.